### PR TITLE
Release documentation updates

### PR DESCRIPTION
**Pull request by dasmanas**
## Description

Fixes pyrsia/pyrsia#

This PR does... by accomplishing... and it can be reviewed by... you can also test the changes by running...

## PR Checklist

- [x] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [x] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/community/get_involved/good_pr.md)
- [x] I've included a good title and brief description along with how to review them.
- [x] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia/issues).

### Code Contributions

- [x] I've run [pre-commit.sh](https://github.com/pyrsia/pyrsia/blob/main/pre-commit.sh) ([pre-commit.bat](https://github.com/pyrsia/pyrsia/blob/main/pre-commit.bat)) successfully.
- [x] I've run the [integrations tests](https://github.com/pyrsia/pyrsia-integration-tests#how-to-set-up-and-run-the-tests)
on the PR branch and all tests passes.

### CODE REVIEW

**Review by efrisch — APPROVED** [review]
👍

**Review by sbtaylor15 — APPROVED** [review]
/lgtm

File: docs/sre/kubernetes-helm.md [hunk 1]
```diff
@@ -22,7 +22,9 @@
 - [Configure Access to Cluster](https://cloud.google.com/sdk/gcloud/reference/container/clusters/get-credentials) - pyrsia-nighty (has spelling mistake) or pyrsia-cluster-1
 
     ```bash
-    gcloud container clusters get-credentials <CLUSTER_NAME>
+    gcloud container clusters get-credentials <CLUSTER_NAME> -z <ZONE>
+    # e.g.
+    gcloud container clusters get-credentials pyrsia-nighty -z us-central1-c
```
Comment (tiainen) [inline docs/sre/kubernetes-helm.md:27]:
```suggestion
    gcloud container clusters get-credentials pyrsia-nightly -z us-central1-c
```

File: docs/tutorials/quick-installation.mdx [hunk 2]
```diff
@@ -201,6 +201,20 @@ sudo apt-get update
...
 
+### Installer Bootstrapping other than default cluster
+
+To connect Pyrsia peer-node authorized boot-node other than default authorize boot node, here is the hack. You have to
+make some edits to `pyrsia.service` file.
+
+```shell
+# E.G. bootstrapping to nightly-cluster.
+sudo sed -i '/ExecStart=/ s/$/ --bootstrap-url http:\/\/boot.nightly.pyrsia.link\/status/' $(sudo find /etc -name pyrsia.service)
+
```
Comment (fishseabowl) [inline docs/tutorials/quick-installation.mdx:212]: This part is not easy for me to understand. I'm confused about `Pyrsia peer-node authorized boot-node`

File: docs/sre/kubernetes-helm.md (hunk 1 above)
Comment (dasmanas) [reply docs/sre/kubernetes-helm.md:27]: Name of our nightly cluster is currently spelled incorrect. Therefore here I kept it like that. 
Screenshot 2023-01-18 at 7 23 11 PM

File: docs/tutorials/quick-installation.mdx (hunk 2 above)
Comment (dasmanas) [reply docs/tutorials/quick-installation.mdx:212]: this is to connect Pyrsia Linux installer to an Authorize node other than `http://boot.pyrsia.link/status`. By providing `--bootstrap-url` in along with `pyrsia_node` we can achieve the same. `find /etc -name pyrsia.service` look for the location of the file and `sed` is appending the additional `--bootstrap-url` argument where it should be in the file.

**Pull request merged**